src/router/logged-out/screens/LanguageScreen.tsx

```diff
@@ -23,7 +23,7 @@ const LanguageScreen = ({ navigation }: Props) => {
         <>
           <Vertical unit={0.5} />
           <CtaButton
-            onPress={() => navigation.navigate('Home')}
+            onPress={() => navigation.navigate('Welcome')}
```

Comment:
This is an important bug fix introduced after the changes for restyling this screen. This button should redirect to the Welcome Screen to start the setup process, not to the Home Screen with the links. Sorry about that.

Reply:
Yeah. I fixed this just earlier. Thankfully I noticed the regression since I was about to publish an upgrade :)